src/Migrations/Version20181121130537.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Doctrine\DBAL\Types\Type;
+
+final class Version20181121130537 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $table = $schema->createTable('user');
+        $table->addColumn('id', Type::GUID);
+        $table->addColumn('email', Type::STRING, ['length' => 100]);
```

Comment:
Is this unique?

Reply:
The email is unique, I add it to the fixture